manifests/t/Timber1900/WebDL/10.3/Timber1900.WebDL.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Timber1900.WebDL
+PackageVersion: 10.3

```

Comment:
Inconsistent PackageVersion, this will cause issues

Reply:
Package version with vXX.X like I had before werent sorting correctly in winget, if I searched for my app version v9.2 showed up like it was the latest version, even tough there were versions v10.X,
Is it possible to change the other manifests versions to remove the v? Or should I post this as an issue?
Thanks for the help.

Reply:
Ahaaa, you can post this as a issue on Winget-cli, because YamlCreate and Windows Explorer sorts it correctly, they probably need to implement Neutral Sorting like i did in YamlCreate
The workaround atm is to delete the other versions.. Or wait for ManifestVersion 1.1.0 which can have a different folder version and PackageVersion.
cc @denelon

Reply:
I fixed the `Inconsistent PackageVersion`, will open a issue in winget-cli about the sorting